src/files.c

```diff
@@ -5059,10 +5082,20 @@ struct monst*
 create_tnnt_npc(x, y)
 xchar x, y;
 {
-    FILE* npcfile = fopen_datafile(TNNT_NPC_FILE, "r", TRUE);
+    FILE* npcfile;
+    int pm_num, gender, npc_level, hpmax;
+    unsigned short mintrinsics;
+    char npcname[BUFSZ], path[BUFSZ];
+    char *npcfilename = pick_npc_file();
     struct monst* npc;
-    if (!npcfile) {
-        impossible("Error opening NPC file '%s' - creating generic mplayer instead", TNNT_NPC_FILE);
+    Sprintf(path, "%s/%s", TNNT_NPC_DIR, npcfilename);
+    if (!*npcfilename || !(npcfile = fopen(path, "r"))) {
+        if (!*npcfilename)
```

Comment:
Hmm, should there be a comment here indicating that in the actual tournament build, we assume that it will be prestocked with npc files?

Reply:
That makes sense to me, it's definitely not obvious and I had to ask about it myself earlier.